relay-protocol/src/lib.rs

```diff
@@ -17,19 +17,23 @@
 )]
 
 mod annotated;
+mod condition;
 mod impls;
 mod macros;
 mod meta;
 mod size;
 mod traits;
+mod utils;
 mod value;
 
 pub use self::annotated::*;
+pub use self::condition::*;
 pub use self::impls::*;
 pub use self::macros::*;
 pub use self::meta::*;
 pub use self::size::*;
 pub use self::traits::*;
+pub use self::utils::*;
```

Comment:
Definitely do not glob-export internal utils to a public namespace. This file contains the function `is_default`, which shouldn't be part of the public API of `relay-protocol`. Instead, we should remove that module and just declare the function locally where needed.

Reply:
I thought about this, you are right that it's not good. Definitely needs to be re-worked.